service/controller/azure/resource_set.go

```diff
@@ -106,6 +106,11 @@ func newResourceSet(config resourceSetConfig) (*controller.ResourceSet, error) {
 			StateGetter: appGetter,
 		}
 
+		c.AllowedAnnotations = []string{
+			"app-operator.giantswarm.io/latest-configmap-version",
```

Comment:
Don't want to bring the latest apiextensions at the moment, so we could use hard-coding annotations.